docs/rst/statistics_backend/get_entities.rst

```diff
@@ -0,0 +1,163 @@
+.. include:: ../exports/alias.include
+
+.. _statistics_backend_get_entities_all:
+
+Get entities of a given kind
+----------------------------
+
+.. warning::
+    This feature is currently not supported.
+    It will be implemented on a future release of *Fast DDS Statistics Backend*.
+
+The |StatisticsBackend-api| singleton can be queried about all the entities of a given |EntityKind-api|.
+For example, |get_entities-api| function can be used to retrieve all the |HOST-api| for which statistics are reported.
+
+.. literalinclude:: /code/StatisticsBackendTests.cpp
+    :language: c++
+    :start-after: //CONF-GET-ENTITIES-ALL-EXAMPLE
+    :end-before: //!
+    :dedent: 8
+
+.. _statistics_backend_get_entities:
+
+Get entities of a given kind related to another entity
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The |StatisticsBackend-api| singleton can be queried about all the entities of a given |EntityKind-api| that are related
+to any entity.
+For example, |get_entities-api| function can be used to retrieve all the |PARTICIPANT-api| running on a given
+|HOST-api|.
+
+.. literalinclude:: /code/StatisticsBackendTests.cpp
+    :language: c++
+    :start-after: //CONF-GET-ENTITIES-EXAMPLE
+    :end-before: //!
+    :dedent: 8
+
+This function returns the related entities according to the following table:
+
+.. list-table:: Entity relations
+   :header-rows: 1
```

Comment:
I believe the last column should be `By Endpoints` instead of `By DomainParticipant`. It's basically the same, but there is no relation between a Participant and a Locator if not through its Endpoints.